frontend/src/components/card/HostActionCard.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import styled from 'styled-components';
+import { SmallActionText } from '../core/Text';
+import { User } from '../../api/User';
+
+const Content = styled.div`
+  // Center div above parent
+  position: absolute;
+  top: 0;
+  left: 50%;
+  transform: translate(-50%, 0);
+  
+  width: 150px;
+  background-color: ${({ theme }) => theme.colors.white};
+  border-radius: 5px;
+  box-shadow: 0 2px 2px rgba(0, 0, 0, 0.24);
+  
+  height: 20px;
+  padding: 10px;
+  
+  // -(height + 2 * padding + :before-border-width / sqrt(2))
+  margin-top: -55px;
+  
+  // Create the arrow at the bottom of the popup
+  &:before {
+    content: "";
+    position: absolute;
+    width: 0;
+    height: 0;
+    left: 50%;
+    box-sizing: border-box;
+    z-index: 1;
+    box-shadow: -2px 2px 2px rgba(0, 0, 0, 0.24);
+    
+    border-style: solid;
+    border-color: transparent transparent ${({ theme }) => theme.colors.white} ${({ theme }) => theme.colors.white};
+    transform-origin: 0 0;
+    transform: rotate(-45deg);
+    
+    border-width: 8px;
+    
+    // -(sqrt(2) * border-width)
+    margin-left: -12px;
+    // -(2 * border-width)
+    bottom: -16px;
```

Comment:
Are these comments still true after taking and updating the room details?

Reply:
Not sure what taking and updating the room details means, but those comments are what I experimented/guessed to be correct (just in case in the future if we wanted to change the size of everything we'd know how to make it look correct)